more_itertools/more.py

```diff
@@ -1202,6 +1203,26 @@ def padded(iterable, fillvalue=None, n=None, next_multiple=False):
             yield fillvalue
 
 
+def repeat_last(iterable):
+    """After the *iterable* is exhausted, keep yielding its last element.
+
+        >>> list(islice(repeat_last(range(3)), 5))
+        [0, 1, 2, 2, 2]
+    """
+    it = iter(iterable)
```

Comment:
I think we can simplify this implementation. What about

```python
item = _marker
for item in iterable:
    yield item

final = default if (item is _marker) else item
yield from repeat(final)
```

Reply:
Didn't notice that there was a marker and didn't want to introduce one. This implementations is much cleaner.